dkpro-core-api-lexmorph-asl/src/main/resources/de/tudarmstadt/ukp/dkpro/core/api/lexmorph/tagset/fr-sequoia-pos.map

```diff
@@ -0,0 +1,30 @@
+*=de.tudarmstadt.ukp.dkpro.core.api.lexmorph.type.pos.POS
+ADJ=de.tudarmstadt.ukp.dkpro.core.api.lexmorph.type.pos.POS_ADJ
+ADJWH=de.tudarmstadt.ukp.dkpro.core.api.lexmorph.type.pos.POS_ADJ
+ADV=de.tudarmstadt.ukp.dkpro.core.api.lexmorph.type.pos.POS_ADV
+ADVWH=de.tudarmstadt.ukp.dkpro.core.api.lexmorph.type.pos.POS_ADV
+CC=de.tudarmstadt.ukp.dkpro.core.api.lexmorph.type.pos.POS_CONJ
```

Comment:
Please add a comment line before each tag line explaining the tag, e.g.

```
# DETWH	   interrogative determiner
DETWH=POS_DET
```

Reply:
I don't have that information. In most cases I was able to conclude which is the coarse-grained major PoS class but the details of what exactly a tag stands for is not available.

Reply:
Please check if the sequoia tagset you have defined is the same as this one:

https://dkpro.github.io/dkpro-core/releases/1.9.0/docs/tagset-reference.html#tagset-fr-melt-pos

At least the tagset mentioned above is used by an OpenNLP POS tagger model created by IXA (see link).